### ISSUE TITLE
Unopenable collection

### ISSUE THREAD

**Issue by andrea-berling** [ankidroid/Anki-Android#6487]
I have a collection that can't seem to be opened on the Android app. It has no problems with  either AnkiWeb or the Linux Anki app though.

The last thing I did before experiencing this problem was changing the type of some notes from the Linux Desktop app and then uploading to AnkiWeb (overwriting the remote collection). The next thing I did was syncing from the Android app (choosing "Download from AnkiWeb"). And now anytime I start the app I get the "Collection not opened" error dialog.

I've tried everything: uninstalling the app, deleting both cache and storage, removing the AnkiDroid directory, reinstalling the app and dowloading from AnkiWeb, uploading a .colpkg from the Linux app to the Android app, uploading the colleciton.anki2 file from the Linux app to the Android app, dumping the collection.anki2 file with sqlite3 (as per FAQ), full sync from server, delete collection and create a new one, check database, nothing. I've had some luck restoring from some backups, but they were still out-of-date w.r.t. AnkiWeb so that wasn't very satisfying.

Unfortunately Anki kept making new backups on each new synch attempt and I ended up loosing all my recoverable backups. I really don't know what else to do.

The net result that I would like to achieve would be to have the Android app downloading the collection that is uploaded to my AnkiWeb account (which is the same that I have on the Linux app) and working. I don't care about overwriting the local app collection, the remote one is the one I want to keep.

If you want I can send you the infamous collection that broke my Android App. Although I'd rather send it directly to a developer than directly posting it here.

###### Reproduction Steps

1. Change note type from desktop App (Linux)
2. Upload to AnkiWeb (Overwriting remote collection)
3. Sync from AnkiDroid (Overwriting local collection)

###### Expected Result

Local collection is the same as the remote collection

###### Actual Result

"Collection not opened", making AnkiDroid basically unusable

###### Debug info

AnkiDroid Version = 2.11.2

Android Version = 9

ACRA UUID = ace6c3b9-97bd-4876-9e47-ca68ad2c3700

###### Research

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by mikehardy** [comment]
I suspect you may be using alpha Anki Desktop versions and be affected the invalid deck config https://github.com/ankidroid/Anki-Android/pull/6455

Definitely try 2.11.3 (just released, or still releasing perhaps - google is slow processing sometimes) and if that doesn't work you might try the current alpha - you can install a side-by-side version as a test: https://github.com/ankidroid/Anki-Android/releases

**Comment by david-allison-1** [comment]
Hi @andrea-berling,

If the above doesn't work, could you provide us with your `collection.anki2` file so we can further diagnose this problem. This is contained in the AnkiDroid folder on your phone.

You can send it to me via email if you don't want to share it publicly. gmail: davidallisongithub

**Comment by andrea-berling** [comment]
Thank you for your response. Apparently updating the Android app + full sync from server + check database sorta did the trick. It still is out of sync w.r.t. AnkiWeb and still asks me whether I want to force upload or download when I sync, but the changes I've made from the Linux app are present on the collection on my phone. I guess I'll study, force upload from the Android app, and then force download from the Linux app hoping this way everything will finally be in sync again. I'll keep you posted

**Comment by david-allison-1** [comment]
@andrea-berling  That's a relief, a "check database" which finds defects will trigger a full sync.

Looks like the cause was #6454 (the only collection bug that I fixed in 2.11.3). 

This is caused by corrupt deck options. Likely on the Anki Desktop side (but we really don't know).

It'd be really useful if you could list your addons for Anki Desktop and/or anything "abnormal" that could have caused this problem.

**Comment by david-allison-1** [comment]
Closing this as the underlying issue is 'resolved', still looking for solutions for the underlying issue.

**Comment by andrea-berling** [comment]
@david-allison-1 Sure. The only addons I use on the Linux app are AnkiConnect (https://ankiweb.net/shared/info/516643804) and Frozen Fields (https://ankiweb.net/shared/info/516643804). As far as Deck Options go, I use the ones recommended by Ali Abdaal in his SkillShare course (https://www.skillshare.com/classes/Learn-Anything-With-Flashcards-The-Ultimate-Guide-To-Anki/1530220171?via=user-profile, lesson "The Best Settings For Anki"), and I remember making a copy of his settings with some minor modifications (which were presenting the cards in the order they were added and to bury related cards) I made to suit a Shared Deck that has cards that increase in difficulty (having them in random order wasn't particularly helpful for learning/studying). I noticed that that particular Options Group disappeared from the Android App after the full sync from the server. Could it have anything to do with the name of the options group? The original is named "Ali Recommended Settings", whereas the modified copy was named "Ali Recommended Settings -random +bury" (or something along those lines).

Lastly I remember doing something that could have caused the corruption, maybe due to a concurrent access to the remote collection in AnkiWeb. Basically after starting the sync process from the Android app and while it was downloading I mistakenly simultaneously started a sync from the Linux app. The Linux app had nothing to push, as it had already synced not five minutes prior, but there was a moment where both apps where trying to access the remote collection. I think I also remember seeing the error message "Only one client can access the collection at one time. Try again later" from the Android app, but I can't be sure.

Hope this can help you. Thank you for your assistance, have a wonderful day.
